time-to-live.md

```diff
@@ -257,7 +257,16 @@ Currently, the TTL feature has the following limitations:
 * The TTL attribute cannot be set on temporary tables, including local temporary tables and global temporary tables.
 * A table with the TTL attribute does not support being referenced by other tables as the primary table in a foreign key constraint.
 * It is not guaranteed that all expired data is deleted immediately. The time when expired data is deleted depends on the scheduling interval and scheduling window of the background cleanup job.
-* For tables that use [clustered indexes](/clustered-indexes.md), if the primary key is neither an integer nor a binary string type, the TTL job cannot be split into multiple tasks. This will cause the TTL job to be executed sequentially on a single TiDB node. If the table contains a large amount of data, the execution of the TTL job might become slow.
+* For tables that use [clustered indexes](/clustered-indexes.md), a TTL job can be split into multiple subtasks only in the following scenarios:
+    - The first column of the primary key or composite primary key is of `INTEGER` or binary string types. The binary string types mainly refer to the following:
+        - `CHAR(N) CHARACTER SET BINARY`
+        - `VARCHAR(N) CHARACTER SET BINARY`
+        - `BINARY(N)`
+        - `VARBINARY(N)`
+        - `BIT(N)`
+    - The character set of the first column of the primary key or composite primary key is `utf8` or `utf8mb4` and the collate is `utf8_bin`, `utf8mb4_bin` or `utf8mb4_0900_bin`.
+    - For tables where the primary key column type is `utf8` or `utf8mb4`, subtasks are split only based on the range of visible ASCII characters. If many primary key values have the same ASCII prefix, it might cause uneven task splitting.
```

Comment:
Character set type?
```suggestion
    - For tables where the character set type of the first column of the primary key is `utf8` or `utf8mb4`, subtasks are split only based on the range of visible ASCII characters. If many primary key values have the same ASCII prefix, it might cause uneven task splitting.
```